src/server/scripts/Northrend/zone_wintergrasp.cpp

```diff
@@ -18,7 +18,7 @@
 #include "ScriptMgr.h"
 #include "Battlefield.h"
 #include "BattlefieldMgr.h"
-#include "BattlefieldWG.h"
+#include "Battlefield/BattlefieldWG.h"
```

Comment:
is the folder needed ?

Reply:
Yes. Only the `scripts` folder is defined as include directory.